.openpublishing.publish.config.json

```diff
@@ -36,38 +36,38 @@
     {
       "path_to_root": "_themes",
       "url": "https://github.com/Microsoft/templates.docs.msft",
-      "branch": "master",
+      "branch": "main",
       "branch_mapping": {}
     },
     {
       "path_to_root": "_themes.pdf",
       "url": "https://github.com/Microsoft/templates.docs.msft.pdf",
-      "branch": "master",
+      "branch": "main",
       "branch_mapping": {}
     },
     {
       "path_to_root": "_csharplang",
       "url": "https://github.com/dotnet/csharplang",
       "branch": "main",
-      "include_in_build": true
+      "branch_mapping": {}
```

Comment:
This needs to be reverted - it's causing build warnings. Same for the 3 changes below this one.